src/e50hx/pyupm_e50hx.i

```diff
@@ -6,8 +6,6 @@
 
 %feature("autodoc", "3");
 
-%include "bacnetmstp.hpp"
-%include "bacnetutil.hpp"
 %include "e50hx.hpp"
```

Comment:
Aren't these needed in the case of a user wanting to access this device using bacnetutil/bacnetmstp in the SWIG languages?